### PR TITLE
Update Bug Report for System Details

### PR DESCRIPTION
**Pull request by Artiume**
Increase priority for system details, constantly have to ask people for their deets. also increased details that should be asked.

**Changes**

**Issues**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -10,6 +10,18 @@ assignees: ''
 **Describe the bug**
 <!-- A clear and concise description of what the bug is. -->
 
+**System (please complete the following information):**
+ - OS: [e.g. Debian, Windows]
```
Comment (dkanada) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
 - OS: [e.g. Debian, Windows, etc.]
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -10,6 +10,18 @@ assignees: ''
...
+ - Virtualization: [e.g. Docker, KVM, etc.]
+ - Clients: [Browser, Android, Fire Stick, etc.]
+ - Browser: [e.g. Firefox, Chrome, Safari]
+ - Jellyfin Version: [e.g. 10.4.3, nightly 20191231]
+ - Playback: [Direct Play, Remux, Direct Stream, Transcode] 
+ - Installed Plugins: [e.g. none, Fanart, Anime, etc.]
+ - Reverse proxy: [e.g. no, nginx, apache, etc.]
```
Comment (dkanada) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
 - Reverse Proxy: [e.g. none, nginx, apache, etc.]
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -10,6 +10,18 @@ assignees: ''
...
+ - Base URL: [e.g. none, yes: /example]
+ - Networking: [e.g. Host, Bridge/NAT]
```
Comment (dkanada) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
 - Networking: [e.g. Host, Bridge, NAT]
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -10,6 +10,18 @@ assignees: ''
...
 <!-- A clear and concise description of what the bug is. -->
 
+**System (please complete the following information):**
+ - OS: [e.g. Debian, Windows]
+ - Virtualization: [e.g. Docker, KVM, etc.]
+ - Clients: [Browser, Android, Fire Stick, etc.]
+ - Browser: [e.g. Firefox, Chrome, Safari]
+ - Jellyfin Version: [e.g. 10.4.3, nightly 20191231]
+ - Playback: [Direct Play, Remux, Direct Stream, Transcode] 
+ - Installed Plugins: [e.g. none, Fanart, Anime, etc.]
```
Comment (telans) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
 - Installed Plugins: [e.g. none, Fanart, Anime]
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -10,6 +10,18 @@ assignees: ''
 **Describe the bug**
 <!-- A clear and concise description of what the bug is. -->
 
+**System (please complete the following information):**
+ - OS: [e.g. Debian, Windows]
+ - Virtualization: [e.g. Docker, KVM, etc.]
+ - Clients: [Browser, Android, Fire Stick, etc.]
+ - Browser: [e.g. Firefox, Chrome, Safari]
+ - Jellyfin Version: [e.g. 10.4.3, nightly 20191231]
+ - Playback: [Direct Play, Remux, Direct Stream, Transcode]
```
Comment (telans) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Is Remux a playback option?

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (telans) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Might want to remove the "etc" here too

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 6]
```diff
@@ -10,6 +10,18 @@ assignees: ''
 **Describe the bug**
 <!-- A clear and concise description of what the bug is. -->
 
+**System (please complete the following information):**
+ - OS: [e.g. Debian, Windows]
+ - Virtualization: [e.g. Docker, KVM, etc.]
```
Comment (telans) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
 - Virtualization: [e.g. Docker, KVM, LXC]
```

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (Artiume) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Technically... It would be the same 😏

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 5 above)
Comment (Artiume) [reply .github/ISSUE_TEMPLATE/bug_report.md]: It isn't defined yet. It's still called out as Playback Streaming  as of right now

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 7]
```diff
@@ -10,6 +10,18 @@ assignees: ''
 **Describe the bug**
 <!-- A clear and concise description of what the bug is. -->
 
+**System (please complete the following information):**
+ - OS: [e.g. Debian, Windows]
+ - Virtualization: [e.g. Docker, KVM, LXC]
+ - Clients: [Browser, Android, Fire Stick, etc.]
+ - Browser: [e.g. Firefox, Chrome, Safari]
```
Comment (JustAMan) [inline .github/ISSUE_TEMPLATE/bug_report.md]: we should ask for browser version, too

Comment (Artiume) [reply .github/ISSUE_TEMPLATE/bug_report.md]: fixed

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 4 above)
Comment (Artiume) [reply .github/ISSUE_TEMPLATE/bug_report.md]: not sure if i want to purge the etc's

**Comment by Artiume** [comment]
Added storage type such as local, NFS or cloud as well

**Pull request merged**